tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -50,18 +50,21 @@ class Swift3WhiskObjectTests
             }
 
             // invoke the action
-            val run = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson))
+            val run = wsk.action.invoke(actionName)
             withActivation(wsk.activation, run, initialWait = 5 seconds, totalWait = 60 seconds) {
                 activation =>
-                    val logs = activation.fields("logs").toString
+                    // should be successful
+                    activation.fields("response").asJsObject.fields("success").toString should be("true")
 
-                    logs should include("It is now")
```

Comment:
Agreed. The log flushing is a stickier issue, that will likely take more time to resolve. I would like to get this merged without waiting for that.